### PR TITLE
New structure for deployments with automated scripts + remove old deployment files

### PR DESCRIPTION
**Pull request by izqui**
Old deployments can be found in the protected [`v0.5` branch](github.com/aragon/deployments/tree/v0.5)

### CODE REVIEW

File: scripts/save [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    throw new Error(`Transaction ${txHash} hasn't been included in a block yet`)
+  }
+
+  const { timestamp } = await web3.eth.getBlock(blockNumber)
+
+  return timestamp
+}
+
+const saveDeploymentEntry = (environmentName, repoName, version, entry) => {
+  const filename = path.join('environments', environmentName, 'deploys.yml')
```
Comment (sohkai) [inline scripts/save]: Might want to preface this with `__dirname` and `../environments` similar to above.

File: scripts/save [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  const repoAddress = await apm.ensResolve(repoName)
+
+  // Only supports one log per transaction
+  const log = receipt.logs.find(
+    ({ address }) => address.toLowerCase() === repoAddress.toLowerCase()
+  )
+
+  const { semanticVersion } = web3.eth.abi.decodeLog(NEW_VERSION_LOG_ABI, log.data, log.topics)
+
+  return apm.getVersion(repoName, semanticVersion, 2000)
```
Comment (sohkai) [inline scripts/save]: I think we can just use the default, or even set a higher one (30s?) for the timeout.

File: scripts/save [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+const getPublishedVersion = async (txHash, repoName, { node, ens }) => {
+  const web3 = new Web3(node)
+  const apm = APM(web3, { ensRegistryAddress: ens })
+
+  const receipt = await web3.eth.getTransactionReceipt(txHash)
+
+  const repoAddress = await apm.ensResolve(repoName)
+
+  // Only supports one log per transaction
+  const log = receipt.logs.find(
```
[the remark this answers is not in the document]
Comment (izqui) [reply scripts/save]: Added in 02c1362

File: scripts/save [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+const NEW_VERSION_LOG_ABI = [{ type: 'uint256', name: 'versionId' }, { type: 'uint16[3]', name: 'semanticVersion'}]
+
+const getArchiveSavePath = (env, repo, version) =>
+ `environments/${env}/${repo}/${repo}@${version}`
+
+const ipfsGetCommand = (hash, output) =>
+  `get -a -C -o ${output} ${hash}`
+
+const getAndSaveIPFSFile = async (hash, output) => {
+  const { stdout } = await execa('ipfs', ipfsGetCommand(hash, output).split(' '))
```
[the remark this answers is not in the document]
Comment (izqui) [reply scripts/save]: If `ipfs get` fails the error is already being printed

**Pull request merged**